x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/ssl/SSLConfigurationReloaderTests.java

```diff
@@ -503,6 +506,27 @@ public void testPEMTrustReloadException() throws Exception {
         assertThat(sslService.sslContextHolder(config).sslContext(), sameInstance(context));
     }
 
+    /**
+     * Tests that the reloader doesn't throw an exception if a file is configured to be outside of the config/ directory.
+     * These errors are handled correctly by the
+     */
+    public void testReadingOutsideConfigDirectoryDoesntFail() throws Exception {
+        Path tempDir = createTempDir();
+        Path clientCertPath = tempDir.resolve("testclient.crt");
+        Settings settings = baseKeystoreSettings(tempDir, null)
+            .putList("xpack.security.transport.ssl.certificate_authorities", clientCertPath.toString())
+            .put("path.home", createTempDir())
+            .build();
+        Environment env = TestEnvironment.newEnvironment(settings);
+        final AtomicReference<Exception> exceptionRef = new AtomicReference<>();
```

Comment:
I think these are no longer needed given the new change of reporting error instead of throwing.